### PR TITLE
Add travis test for indigo

### PR DESCRIPTION
**Pull request by wkentaro**

### CODE REVIEW

**Comment by wkentaro** [comment]
I tried on my repo. https://github.com/wkentaro/image_pipeline/pull/1

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  - sudo rosdep init
+  - rosdep update
+  # Make a workspace
+  - mkdir -p /tmp/image_pipeline_ws/src
+  # Add this folder to the workspace
+  - ln -s $(pwd) /tmp/image_pipeline_ws/src/image_pipeline
+  # Install dependencies
+  - cd /tmp/image_pipeline_ws/src
+  - git clone https://github.com/ros-perception/vision_opencv.git
+  - (cd vision_opencv && git checkout 7ff3fa72ac05377b9d7c8511ef9378f0e50eff82)
```
Comment (vrabaud) [inline .travis.yml]: you are checking out a specific version of vision_opencv

Comment (wkentaro) [inline .travis.yml]: actually, the build fails with this change. https://github.com/ros-perception/vision_opencv/commit/2be3371fa6f793b9562f0f891fe561effd511ca9

by saying 

```
[cv_bridge] In file included from /home/wkentaro/ros/indigo/src/ros-perception/vision_opencv/cv_bridge/src/cv_bridge.cpp:36:0:                                 
[cv_bridge] /home/wkentaro/ros/indigo/src/ros-perception/vision_opencv/cv_bridge/src/boost/endian/conversion.hpp:12:47: fatal error: boost/predef/detail/endian_compat.h: No such file or directory 
[cv_bridge]  #include <boost/predef/detail/endian_compat.h>                                                                                                    
[cv_bridge]                                                ^                                                                                                   
[cv_bridge] compilation terminated.                                                                                                                            
[cv_bridge] make[2]: *** [src/CMakeFiles/cv_bridge.dir/cv_bridge.cpp.o] Error 1      
```

Comment (wkentaro) [inline .travis.yml]: so I checked out specific version in the test.

Comment (vrabaud) [inline .travis.yml]: ok, but the travis file should be agnostic to that. vision_opencv should be installed from deb (and very sorry about the error, I finally got a dev job that was failing (I was waiting for the dev job to be done before releasing)). I am on it.

Comment (wkentaro) [inline .travis.yml]: okay, so I will remove that lines.

**Comment by vrabaud** [comment]
let's try it out, thx !

**Pull request merged**